internal/index_read.go

```diff
@@ -156,6 +150,9 @@ func readStats(r io.Reader, typ string) (*ReferenceStats, error) {
 	return &stats, nil
 }
 
+// chunkSize determines the number of offsets  consumed by each binary.Read.
```

Comment:
I moved it inside and squashed. vOffs must be a slice so it can be truncated on the final loop.